content/tbd/1000 Landscapes for 1 billion people.md

```diff
@@ -0,0 +1,53 @@
+---
+created: 2023-11-30
+contributors:
+  - Catherine Tran
+title: 1000 Landscapes for 1 Billion People
+url: https://landscapes.global/
+locations:
+  - International network
+start: 2018
+end: 
+size: 4-10
+image: assets/img/1000-landscapes-for-1-billion-people-homepage.png
+sectors: 
+activities: 
+cohere: true
+metadata_version: "1"
+---
+
+# About
```

Comment:
Let's leave out the about heading. We just lead straight in with the description.

Reply:
ok sure